.github/renovate/special/customVersioning.json5

```diff
@@ -16,88 +16,90 @@
     {
       "description": ["Custom versioning for minio"],
       "matchDatasources": ["docker"],
-      "matchPackagePatterns": ["minio"],
+      "matchPackagePatterns": ["docker.io/minio/minio"],
       "versioning": "regex:^RELEASE\\.(?<major>\\d+)-(?<minor>\\d+)-(?<patch>\\d+)T.*Z$"
     },
     {
       "matchDatasources": ["docker"],
-      "matchPackagePatterns": ["kopia"],
+      "matchPackagePatterns": ["docker.io/kopia/kopia"],
       "allowedVersions": "<999"
     },
     {
       "matchDatasources": ["docker"],
-      "matchPackagePatterns": ["postgresql"],
+      "matchPackagePatterns": ["ghcr.io/cloudnative-pg/postgresql"],
       "allowedVersions": "<17"
     },
```

Comment:
Please limit yourself to things in charts, this is a COMPLETELY different package